h/admin/views/features.py

```diff
@@ -52,8 +52,11 @@ def cohorts_index(context, request):
 def cohorts_add(request):
     """Create a new feature cohort."""
     cohort_name = request.params['add']
-    cohort = models.FeatureCohort(name=cohort_name)
-    request.db.add(cohort)
+    if cohort_name not in models.FeatureCohort.RESERVED_FEATURECOHORT_NAMES:
```

Comment:
Couple of different things to say to this:

1. Don't worry about it -- it's an admin interface. If people create a cohort with a silly name we can address that by communication rather than code.
2. If you did want to do this it would probably be better to implement it further back (i.e. on the model, using a [validator](http://docs.sqlalchemy.org/en/latest/orm/mapped_attributes.html#simple-validators)) rather than in the view. (Although the view would need to know how to respond to a validation error.)
3. Namespaces are great! If you're storing a set of reserved names on the `FeatureCohort` class, then it's clear they refer to reserved names for feature cohorts, so you can just call it `RESERVED_NAMES` rather than `RESERVED_FEATURECOHORT_NAMES`.

In summary, though, I don't think this is necessarily needed s